scaleway/resource_instance_server.go

```diff
@@ -270,6 +270,11 @@ func resourceScalewayInstanceServer() *schema.Resource {
 			"organization_id": organizationIDSchema(),
 			"project_id":      projectIDSchema(),
 		},
+		CustomizeDiff: customizeDiffLocalityCheck(
```

Comment:
Having a sequence would make it easier to read here. https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/customdiff#Sequence